app/views/projects/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1><%= @project.name %></h1>
+<h1>Material: <%= @project.material %></h1>
+<%
+=begin%>
+ <h1>Theme: <%= @challenge.theme %></h1> 
```

Comment:
I think this is a similar issue to what was happening above -- how can we get to a challenge from just a project? The answer there is to start with your **associations** on the model you're beginning with. Since we have a `@project` instance, what associations does the Project model have? 

It has a `belongs_to :challenge` line. Remember that the `belongs_to` and `has_many` keywords act like `attr_reader`s in that they give us a method to access the instances that are related to an object. Using those, we can call 
```ruby
@project.challenge
```

and we'd get the challenge instance that this project belongs_to. Try it out!

Reply:
Once we have the `@project.challenge`, we'd have a challenge instance which would give us access to all its attributes and methods, so we could call `@project.challenge.theme` on line 5 of your view, and that's all we need!